### PR TITLE
scripts to test reflector

### PR DESCRIPTION
**Pull request by jobevers**
@lyoshenka this is what I've been using to test reflector recently.

Something like
`python download_blobs_from_reflector.py 198.199.87.180 --batch=10 --size=50 --timeout=300`

will try to download 50 blobs in 5 batches of 10.

### CODE REVIEW

**Comment by lyoshenka** [comment]
looks good. when you have a moment, could you add a flag to pass in the ssh key to use when SSHing?

File: scripts/download_blobs_from_reflector.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import json
+import random
+import subprocess
+import sys
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('reflector_ip')
+    parser.add_argument('--ssh-key')
```
Comment (jobevers) [inline scripts/download_blobs_from_reflector.py]: @lyoshenka the script does have a flag to pass in a key.

**Comment by lyoshenka** [comment]
you read my mind!

**Comment by lyoshenka** [comment]
@jobevers forgot to say - LGTM

**Comment by jobevers** [comment]
The failing check was from circle, which wasn't setup to actually run anything.

**Pull request merged**